src/screens/KeyBasedAssetTransfer/KeyBasedAssetTransferStatus.js

```diff
@@ -164,7 +164,7 @@ const KeyBasedAssetTransferStatus = ({
         <ScrollWrapper contentContainerStyle={{ paddingVertical: spacing.large }}>
           <Wrapper flex={1} center regularPadding>
             <Animation source={animationSuccess} />
-            <Title center>Key based assets migration complete</Title>
+            <Title center>{t('transactions.keyWalletAssetsMigrationDone')}</Title>
```

Comment:
It's
```suggestion
            <Title center>{t('transactions.title.keyWalletAssetsMigrationDone')}</Title>
```
in the json file.

Reply:
Fixed. Thanks for spotting it! :)